### DISCUSSION TITLE
Bring back #AlphanetToday type network Stats ? - Tweet a Monthly/bi-monthly template.

### DISCUSSION THREAD

**Discussion by coinselor** [coinselor/twitter#11] [Tweet Ideas]
Similar to:
 
 https://t.co/bSJKWez6uk
 
 
![FYNJGSQWYAATkS2](https://github.com/coinselor/twitter/assets/92316026/f5c6aa58-92ff-4c9c-8db6-1378dec55140)

**Comment by trevorleazy** [comment]
Seems like an easy bit of consistent content, reuse the graphics and update the stats.
